src/runtime/virtcontainers/container.go

```diff
@@ -767,6 +773,25 @@ func newContainer(ctx context.Context, sandbox *Sandbox, contConfig *ContainerCo
 		ctx:           sandbox.ctx,
 	}
 
+	// Set the Annotations of SWAP to Resources
+	if resourceSwappinessStr, ok := c.config.Annotations[resourceSwappinessLabel]; ok {
+		resourceSwappiness, err := strconv.ParseUint(resourceSwappinessStr, 0, 64)
+		if err == nil && resourceSwappiness > 200 {
+			err = fmt.Errorf("swapiness should not bigger than 200")
+		}
+		if err != nil {
+			return &Container{}, fmt.Errorf("Invalid container configuration Annotations %s %v", resourceSwappinessLabel, err)
+		}
+		c.config.Resources.Memory.Swappiness = &resourceSwappiness
+	}
+	if resourceSwapInBytesStr, ok := c.config.Annotations[resourceSwapInBytesLabel]; ok {
+		resourceSwapInBytes, err := strconv.ParseInt(resourceSwapInBytesStr, 0, 64)
```

Comment:
This looks unsafe - what if `io.katacontainers.resource.swap_in_bytes = "-1"`? `ParseUint()`?

Reply:
Fixed.